lib/SampleService/core/config.py

```diff
@@ -105,11 +122,18 @@ def build_samples(config: Dict[str, str]) -> Tuple[Samples, KBaseUserLookup, Lis
             workspace-read-admin-token: [REDACTED FOR YOUR ULTIMATE PLEASURE]
             kafka-bootstrap-servers: {kafka_servers}
             kafka-topic: {kafka_topic}
+            metadata-validators-config-repo: {metaval_repo}
             metadata-validators-config-url: {metaval_url}
     ''')
 
     # build the validators before trying to connect to arango
-    metaval = get_validators(metaval_url) if metaval_url else MetadataValidatorSet()
+    metaval = get_validators(
+        repo_path=metaval_repo,
+        repo_file=metaval_filename,
+        prerelease_ok= (metaval_prelease_ok or '').lower() in ('true', 'yes', 'y', '1'),
+        url=(metaval_url or None),
+        token=(github_token or None)) if (
+            metaval_url or metaval_repo) else MetadataValidatorSet()
```

Comment:
IMO this would be a lot easier to read with a standard if block (this incorporates all the other suggestions above)

```suggestion
    if metaval_url or metaval_repo:
        metaval = get_validators(
            repo_path=metaval_repo,
            repo_file=metaval_filename,
            prerelease_ok=(metaval_prelease_ok or '').lower() in ('true', 'yes', 'y', '1'),
            url=metaval_url
            token=github_token
        )
    else:
        metaval = MetadataValidatorSet()
```

Reply:
Also I'd handle the boolean checking in a specific method for booleans like `_check_bool` mentioned above

Reply:
:+1: